src/downshift.js

```diff
@@ -658,12 +663,10 @@ class Downshift extends Component {
     this.inputId = firstDefined(this.inputId, rest.id, `${this.id}-input`)
     let onChangeKey
     /* istanbul ignore next (preact) */
-    if (preval`module.exports = process.env.BUILD_PREACT === 'true'`) {
+    if (this.props.isPreact) {
```

Comment:
I'm sorry, but doing this will prevent these statements from being eliminated during uglification resulting in a bigger bundle. Please restore all of these to what they were before.

Reply:
👍